sphinx_design/badges_buttons.py

```diff
@@ -195,7 +192,6 @@ def run(self) -> List[nodes.Node]:
 
         # `visit_reference` requires that a reference be inside a `TextElement` parent
         container = nodes.paragraph(classes=self.options.get("align", []))
-        self.set_source_info(container)
```

Comment:
This set_source_info breaks translations, but the one in "expand" for the grid_container doesn't.